travis.sh

```diff
@@ -12,3 +12,14 @@ make -f posix.mak all DMD="$(which dmd)"
 make -f posix.mak test DMD="$(which dmd)" \
     RDMD_TEST_COMPILERS=dmd,"$LDMD2" \
     VERBOSE_RDMD_TEST=1
+
+# Test setup.sh
+dmd=dmd/generated/linux/release/64/dmd
+dir=generated/setup.sh-test
+rm -rf "$dir" && mkdir "$dir" && pushd "$dir"
+echo "y" | ../setup.sh
+echo 'void main(){ import std.stdio; "Hello World".writeln;}' | "./${dmd}" -run - | grep -q "Hello World"
+popd && rm -rf "$dir" && mkdir "$dir" && pushd "$dir"
+echo "y" | ../setup.sh --tag=2.078.1
+echo 'void main(){ import std.stdio; __VERSION__.writeln;}' | "./2.078.1/${dmd}" -run - | grep -q "2078"
+popd
```

Comment:
Poor man's sanity check, but if these work, chances are that the setup was successful.